.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
```

Comment:
I think daily is too noisy and ends up providing us with more notifications than needed, can you make it weekly or monthly so that all notifications arrive at the same time.

Reply:
We actually have "daily" in the docker-stacks.
Are you sure we want to have different values or should we adjust this there as well?

Reply:
If you want to ensure they are kept in sync, I'd appreciate if they are both adjusted to have a lower frequency.

I'm trying to avoid the notification noise, and daily updates to github action versions isn't justified having daily updates. I've been configuring very common updates in the past and ended up lowering the frequency over time realizing it is becoming problematic.

Reply:
Thanks for your work to keep things in sync, which I think is a good practice, and your general thorough maintenance!

Reply:
Done "weekly"